src/components/ebay-textbox/index.js

```diff
@@ -38,9 +43,11 @@ module.exports = require('marko-widgets').defineComponent({
 
 function forwardEvent(eventName) {
     return function(originalEvent, el) {
-        emitAndFire(this, `textbox-${eventName}`, {
-            originalEvent,
-            value: (el || this.el.querySelector('input, textarea')).value
-        });
+        if (!this.state.disabled) {
```

Comment:
I'm confused as to why these events would get forwarded in the first place if the DOM element is disabled.

Reply:
Essentially, our tests “fake” the click, focus, blur, etc. If I were to try and write the test for a disabled state without this check it would always fail.

Should I remove the test in order to remove this check?

Reply:
Thanks for explaining. I don't think we should be adding logic to our code purely for the purpose of our test setup ... if that makes sense.